cmake.deps/cmake/BuildLuajit.cmake

```
@@ -30,7 +30,7 @@ function(BuildLuajit)
   if(UNIX)
     add_custom_command(
       TARGET ${_luajit_TARGET}
-      COMMAND ${CMAKE_COMMAND} -E create_symlink luajit-2.1.0-beta3 ${DEPS_BIN_DIR}/${_luajit_TARGET})
+      COMMAND ${CMAKE_COMMAND} -E create_symlink luajit-2.1 ${DEPS_BIN_DIR}/${_luajit_TARGET})
```

This makes the build work for me locally, although it's ugly. I think the dot comes from a luajit bug in the build process, so might be worth waiting a bit to confirm this so we can fix this upstream.

```suggestion
      COMMAND ${CMAKE_COMMAND} -E create_symlink luajit-2.1. ${DEPS_BIN_DIR}/${_luajit_TARGET})
```